langchain4j-anthropic/pom.xml

```
@@ -108,6 +108,18 @@
                     <skipTests>${skipAnthropicITs}</skipTests>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>${maven-jar-plugin.version}</version>
+                <configuration>
+                    <archive>
+                        <manifestEntries>
+                            <Automatic-Module-Name>dev.langchain4j.model.anthropic</Automatic-Module-Name>
```

```suggestion
                            <Automatic-Module-Name>dev.langchain4j.anthropic</Automatic-Module-Name>
```
To be consistent with other modules